tools/SourceKit/lib/SwiftLang/SwiftSourceDocInfo.cpp

```diff
@@ -10,12 +10,13 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include "SwiftASTManager.h"
-#include "SwiftLangSupport.h"
 #include "SourceKit/Support/FileSystemProvider.h"
 #include "SourceKit/Support/ImmutableTextBuffer.h"
 #include "SourceKit/Support/Logging.h"
 #include "SourceKit/Support/UIdent.h"
+#include "SwiftASTManager.h"
+#include "SwiftEditorDiagConsumer.h"
+#include "SwiftLangSupport.h"
```

Comment:
Nit: We usually sort implementation headers outside include directories to the top of the list, like it was before.

Reply:
I just accepted what `git-clang-format` offered. Do you think it’s better to keep the old sorting or give in to `clang-format`?

Reply:
Personally I would keep the old sorting, but I'm not worried about it either way if you disagree